README.md

```diff
@@ -180,6 +300,84 @@ Localization of the editor is an ongoing effort, the main target usage of embedd
+* `https://data.vlaanderen.be/id/concept/BesluitType/7d95fd2e-3cc9-4a4c-a58e-0fbc408c2f9b`
+* `https://data.vlaanderen.be/id/concept/BesluitType/3bba9f10-faff-49a6-acaa-85af7f2199a3`
+* `https://data.vlaanderen.be/id/concept/BesluitType/0d1278af-b69e-4152-a418-ec5cfd1c7d0b`
+* `https://data.vlaanderen.be/id/concept/BesluitType/e8afe7c5-9640-4db8-8f74-3f023bec3241`
+* `https://data.vlaanderen.be/id/concept/BesluitType/256bd04a-b74b-4f2a-8f5d-14dda4765af9`
+* `https://data.vlaanderen.be/id/concept/BesluitType/67378dd0-5413-474b-8996-d992ef81637a`
+
+If this condition is met you will find a button called `Voeg mobiliteitsmaatregel in` in the insert menu. When this button is clicked a modal will apear where you can filter and select which roadsign regulation you want to insert.
+
+![roadsign regulation modal](https://i.imgur.com/z7My8lm.png)
+
+### Variable
+This plugin allows you to insert variables in the document, a variable is a value to be filled by another user, ideally you will have 2 separate instances of the editor, the first one will enable this plugin and the other will enable the template-variable plugin.
+In order to insert a variable you will need to use the insert variable card.
+
+![insert variable card](https://imgur.com/9kSqgXc.png)
+
+In this dropdown you can select one of the 4 default types of variables (or more if you added your custom variables to the config): text, number, date or codelist. If codelist is selected you will need to specify which codelist you want to use
+
+### Template variable
+This plugin allows to fill the codelist variables generated by the variable plugin, if you are in one of these variables the following card will appear:
+
+![template variable card](https://imgur.com/b5Kmmqj.png)
+
+Using this card you will be able to select one of the codelist values.
+
+### Article Structure
+This plugin is in charge of inserting and manipulating structures, for inserting the structures you will find the buttons on the insert menu of the right sidebar, for example if you are in besluit mode you will only have a button that says `Insert article`, if you are in regulatory statement mode you will be able to insert titles, chapters, sections...
+After inserting a structure you will be presented with a card, where you can move up and down the structure or delete it. For deleting you have 2 options, deleting just the structure (if possible) trying to upwrap the content or deleting the structure with its content.
+
+Note that all of these buttons can be disabled if the action is not possible.
+
+![article structure card](https://imgur.com/2zkbNw3.png)
+
+### Table of Contents
+This plugin provides a toggle in the top bar that says `Show Table of Contents` if you click this toggle the table of contents will appear, take into account that the table of contents works with the regulatory statement mode of the article structure by default but this can be modified.
```

Comment:
```suggestion
This plugin provides a toggle in the top bar that says `Show Table of Contents`. If you click this toggle the table of contents will appear. Take into account that the table of contents works with the regulatory statement mode of the article structure by default but this can be modified.
```

modified how? -> either we say it only works for reg statements, or we provide an example config on how to make it work for decisions. They can't adjust the schema, so they don't know which nodes are relevant.

Reply:
Yeah true, I think it's better if we tell them not to modify it for now, but we should look for a way of creating custom nodes in embeddable